### ISSUE TITLE
Bug with vehicle in SA:MP 0.3.7 R1

### ISSUE THREAD

**Issue by mrreznitsky** [Whitetigerswt/gtasa_crashfix#71]
When another player kills me and I sit in the car, I just fall out of it and do not respawn until the other player gets into the car from which I fell out.

**Comment by Vahinkolaukausosuma** [comment]
yeah and when u kill(headshot?) an npc in a car, they fall out, get glued to the car, and I CANT enter the car ever again xd
CrashFix_Misc21

**Comment by juanpenia** [comment]
This, I know you're not going to maintain this mod but I just noticed this today. I had the same problem with npcs, and I thought it was a game bug, but seems like not because I just tested it.

**Comment by uBlock-user** [comment]
https://github.com/Whitetigerswt/gtasa_crashfix/issues/34
